### DISCUSSION TITLE
Nextflow crashed while running the viralrecon pipeline

### DISCUSSION THREAD

**Discussion by faiksohail** [nextflow-io/nextflow#4476] [Q&A]
Hi,

I am using nextflow to run a pipeline (nf-core/viralrecon)

when I run the command - `nextflow run nf-core/viralrecon -profile singularity --max_memory '12.GB' --max_cpus 4 --input samplesheet.csv --outdir results/viralrecon --protocol amplicon --genome 'MN908947.3' --primer_set artic --primer_set_version 3 --skip_kraken2 --skip_assembly --skip_pangolin --skip_nextclade --platform illumina --fasta 'MN908947.fasta' --primer_bed MN908947.bed`

first warnings -
**WARNING** : Could not load nf-core/config profiles: https://raw.githubusercontent.com/nf-core/configs/master/nfcore_custom.config

**WARNING** : Could not load nf-core/config/viralrecon profiles: https://raw.githubusercontent.com/nf-core/configs/master/pipeline/viralrecon.config ```

Then, after execution it says - 
Downloading plugin nf-amazon@1.11.0

**WARN** : Unable to start plugin 'nf-amazon' required by s3://ngi-igenomes/test-data/viralrecon/kraken2_human.tar.gz

Missing plugin 'nf-amazon' required to read file: s3://ngi-igenomes/test-data/viralrecon/kraken2_human.tar.gz

-- Check script '/root/.nextflow/assets/nf-core/viralrecon/./workflows/illumina.nf' at line: 26 or see '.nextflow.log' file for more details

I am using
```
 N E X T F L O W
      version 22.10.1 build 5829
      created 27-10-2022 18:39 UTC (28-10-2022 00:09 IDT)
      cite doi:10.1038/nbt.3820
      http://nextflow.io
```

Please help in addressing the issue at the earliest.

Thanks

**Comment by bentsherman** [comment]
Could you try again with Nextflow 23.10.0 ?

**Reply by faiksohail** [reply]
Running with the same issue with the latest Nextflow version **23.10.0** as well.

 I would be happy to share any of the details in order to resolve this issue at the earliest.

Thanks
